build/webpack.config.js

```diff
@@ -71,8 +72,10 @@ webpackConfig.plugins = [
 if (__DEV__) {
   debug('Enable plugins for live development (HMR, NoErrors).');
   webpackConfig.plugins.push(
+    new CopyWebpackPlugin([{ from: paths.client('static') }]),
     new webpack.HotModuleReplacementPlugin(),
     new webpack.NoEmitOnErrorsPlugin()
+
```

Comment:
**Q:** How is adding this plugin different from the [existing code](https://github.com/teamleadercrm/teamleader-ui/blob/component/avatar/bin/compile.js#L18-L20)?

Reply:
It enables me to serve the 'static' folder on localhost, so I could load an image from http://localhost/avatars/1.png

Reply:
That's the whole point of the existing code, works perfectly here...